app/src/main/res/values/strings.xml

```diff
@@ -7,4 +7,5 @@
     <string name="login">Login</string>
     <string name="noAccountYet"><u>No account yet?\nCreate one</u></string>
     <string name="LoginFailed">Login failed</string>
+    <string name="RegisterFailed">Register failed</string>
```

Comment:
Lets stay consistent with cammelCase or other style not both

Reply:
yes